src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationService.java

```diff
@@ -78,6 +80,15 @@ private DiseaseAnnotation upsertAnnotation(DiseaseModelAnnotationDTO annotationD
         }
         annotation.setNegated(annotationDTO.getNegation() == DiseaseModelAnnotationDTO.Negation.not);
 
+        if (CollectionUtils.isNotEmpty(annotationDTO.getWith())) {
+            List <Gene> withGenes = new ArrayList<>();
+            annotationDTO.getWith().forEach(with -> {
+                Gene withGene = geneDAO.find(with);
```

Comment:
find() is ment to work with the primary key on the table. So in this case with.getCurie().

Reply:
`with` is just a string, so I've switched it to `geneDAO.getByIdOrCurie(with)`

Reply:
I'm sorry.. I didn't catch that... the getWith from the DTO is a string... ok.